scripts/issues.coffee

```diff
@@ -0,0 +1,43 @@
+# Description:
+#   Create GitHub issues for CH beheer and PC.COM. Please be concise.
+#
+# Dependencies:
+#   octonode
+#
+# Configuration:
+#   HUBOT_GITHUB_TOKEN = <accessToken>
+#
+# Commands:
+#   @pccom <query> - Creates a new issue and tags it 'pccom'
+#   @beheer <query> - Creates a new issue and tags it 'beheer'
+
+github = require 'octonode'
+
+githubToken = process.env.HUBOT_GITHUB_TOKEN;
+
+issue = null
```

Comment:
Dat heb ik dus geprobeerd, en dat is niet waar geloof ik. Zie https://github.com/github/hubot/blob/master/src/listener.coffee. `response` ziet er zo uit: 
``` javascript
response: {
  robot: Object,
  match: Boolean,
  message: Object
}
```